tests/doc_test.py

```diff
@@ -8,11 +9,10 @@
     reason="Python 3 doesn't have the UnicodeEncodeError problem",
 )
 def test_docs():
-    from doctest import testfile, ELLIPSIS
-    failures, _ = testfile(
+    failures, _ = doctest.testfile(
```

Comment:
pytest has built-in doctesting -- can probably just delete this and use that

Reply:
Cool. Converted to `pytest --doctest-glob`.

I do lose the skip reason though. If I do an inline `pytest.skip`, it will skip the entire file in addition to cluttering the docstrings with pytest specific info.